packages/oas-utils/src/transforms/import-spec.ts

```diff
@@ -72,8 +72,14 @@ export const importSpecToWorkspace = async (spec: string | AnyObject) => {
         cookies: {},
       }
 
+      // An operation can have component level parameters as well :)
+      const pathAndOperationParameters = [
+        ...path.parameters,
+        operation.parameters,
```

Comment:
U want the ... Here?

Reply:
@amritk yeah i actually wanted it in both! but i forgot to commit it up :despair:

all fixed now, theyre both arrays of objects. but sometimes both are null